spec/lib/formatter_spec.rb

```diff
@@ -107,9 +107,13 @@ module CodeClimate::TestReporter
     end
 
     describe '#short_filename' do
+      def call(*args)
+        formatter.send(:short_filename, *args)
+      end
+
       it 'should return the filename of the file relative to the SimpleCov root' do
-        expect(formatter.short_filename('file1')).to eq('file1')
-        expect(formatter.short_filename("#{::SimpleCov.root}/file1")).to eq('file1')
+        expect(call('file1')).to eq('file1')
+        expect(call("#{::SimpleCov.root}/file1")).to eq('file1')
```

Comment:
un-dry-ed it

On Sat, Apr 18, 2015 at 7:27 AM, Noah Davis <notifications@github.com>
wrote:

> In spec/lib/formatter_spec.rb
> <https://github.com/codeclimate/ruby-test-reporter/pull/84#discussion_r28644407>
> :
>
> >        it 'should return the filename of the file relative to the SimpleCov root' do
> > -        expect(formatter.short_filename('file1')).to eq('file1')
> > -        expect(formatter.short_filename("#{::SimpleCov.root}/file1")).to eq('file1')
> > +        expect(call('file1')).to eq('file1')
> > +        expect(call("#{::SimpleCov.root}/file1")).to eq('file1')
>
> I think I prefer the explicit style. This is harder for me to read.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeclimate/ruby-test-reporter/pull/84/files#r28644407>
> .
>